Composer/packages/client/src/pages/design/DebugPanel/TabExtensions/WatchTab/WatchTabObjectValue.tsx

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { css, jsx } from '@emotion/core';
+import { JsonEditor } from '@bfc/code-editor';
+import React, { useMemo } from 'react';
+import { useRecoilValue } from 'recoil';
+
+import { userSettingsState } from '../../../../../recoilModel';
+
+const editorStyles = css`
+  border: none;
+`;
+
+const objectCellStyle = (numLinesOfJson: number) => css`
+  height: ${numLinesOfJson * 18}px;
```

Comment:
@tonyanziano On a huge JSON like the dialog object we would need to cap the height of the cell. Basically, allowing scroll to inside the editor. Can we confirm the max height of the value cell @mewa1024 
![Screen Shot 2021-06-28 at 3 47 06 PM](https://user-images.githubusercontent.com/13004779/123712918-28384580-d828-11eb-8b29-df2abcea19eb.png)

Reply:
Great point!

Reply:
What about 80px max height? That would show 4 full lines, plus the top of a 5th line, if any. (Are we assuming that if it's an enormous JSON object the user would copy and paste into a code editor to view the whole thing?)

Reply:
Yep.